.gitignore

```diff
@@ -39,3 +39,8 @@ test/__pycache__/
 **/.pymodhis
 /build/
 /dist/
+examples/functional
+examples/twisted
```

Comment:
functional, twisted, gui and tools are gone,

please see:
https://github.com/riptideio/pymodbus/commit/329dea9ff5abd40f2985dcf3f7dd9c29049f1189

Reply:
Yes, this is a reminiscent from my local setup. I will remove them